marklogic-data-hub-central/ui/src/components/modeling/graph-view/graph-view.tsx

```diff
@@ -22,16 +22,50 @@ type Props = {
 const GraphView: React.FC<Props> = (props) => {
 
   const {modelingOptions, setSelectedEntity, closeSidePanelInGraphView} = useContext(ModelingContext);
+  const [nodeSuggested, setNodeSuggested] = useState(["a"]);
+  const [entitySelected, setEntitySelected] = useState("");
+  const [isEntitySelected, setIsEntitySelected] = useState(false);
 
   const publishIconStyle: CSSProperties = {
     width: "20px",
     height: "20px",
     fill: "currentColor",
   };
 
+  const handleFocus = () => {
+    setNodeSuggested([]);
+  };
+
+  const handleTypeaheadChange = (value: any) => {
+    setEntitySelected(value);
```

Comment:
Are these values specific to the filtering feature? If so, can we call them something else to distinguish them from entities selected by a user click? Maybe "entityFiltered" and "isEntityFiltered"?